test/integration/ies.go

```diff
@@ -0,0 +1,115 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
this file looks like a leftover ?

Reply:
yes. I forgot to sync with the main branch. Solved